src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -78,110 +75,116 @@ export const ArtistAuctionResultItem: React.FC<Props> = props => {
       width="100%"
       onClick={onAuctionResultClick}
     >
-      <GridColumns>
-        <Column span={[4, 2]}>
-          <ResponsiveBox
-            aspectWidth={1}
-            aspectHeight={1}
-            maxWidth={130}
-            bg="black10"
-          >
-            {image ? (
-              <Image
-                src={image.src}
-                srcSet={image.srcSet}
-                width="100%"
-                height="100%"
-                alt=""
-                lazyLoad
-              />
-            ) : (
-              <Box
-                display="flex"
-                justifyContent="center"
-                alignItems="center"
-                height="100%"
-              >
-                <NoArtIcon height={24} width={24} fill="black60" />
-              </Box>
-            )}
-          </ResponsiveBox>
-        </Column>
+      <RouterLink
+        to={`/auction-result/${internalID}`}
+        textDecoration="none"
+        style={{ pointerEvents: user ? "auto" : "none" }}
+      >
+        <GridColumns>
+          <Column span={[4, 2]}>
+            <ResponsiveBox
+              aspectWidth={1}
+              aspectHeight={1}
+              maxWidth={130}
+              bg="black10"
```

Comment:
Changes in this file displayed super weirdly. Actually we just wrapped `<GridColumns />` in `<RouterLink />`